WalletWasabi.Fluent.Desktop/WalletWasabi.Fluent.Desktop.csproj

```diff
@@ -29,7 +29,7 @@
 		<NeutralLanguage>en-US</NeutralLanguage>
 		<AssemblyTitle>Wasabi Wallet Fluent</AssemblyTitle>
 		<Description>
-			Open-source, non-custodial, privacy focused Bitcoin wallet for Windows, Linux, and Mac. Built-in Tor, CoinJoin, PayJoin and coin control features.
+			Open-source, non-custodial, privacy focused Bitcoin wallet for Windows, Linux, and Mac. Built-in Tor, coinjoin, PayJoin and coin control features.
```

Comment:
What about Payjoin here?

Reply:
@yahiheb see #7091 if you want to also change that. I didn't dare to include it in the same PR as the other fixes